articles/libraries/lock/v10/migration-guide.md

```diff
@@ -16,8 +16,10 @@ The following instructions assume you are migrating from **Lock 9** to the lates
 ### Customizing Options
 - The `show` method no longer takes any arguments. You pass the options to the constructor and you listen for an `authenticated` event instead of providing a callback. You can listen for this event with the [on](/libraries/lock/v10/api#on-event-callback-) method.
 
-### Authenticated Event
-- The `authenticated` event listener has a single argument, an `authResult` object. This object contains the following properties: `idToken`, `accessToken`, `state`, `refreshToken` and `idTokenPayload`. Most of them correspond to the arguments passed to the `show` method's callback.
+### Events Changed
+- Events have significantly changed between Lock 9 and Lock 10. The events that were emitted [in Lock 9](/libraries/lock/v9/events) are no longer used in Lock 10. The new events list for Lock 10 can be found on the [Lock 10 API page](/libraries/lock/v10/api).
+- Important notes about the new 'authenticated' event: The `authenticated` event listener has a single argument, an `authResult` object. This object contains the following properties: `idToken`, `accessToken`, `state`, `refreshToken` and `idTokenPayload`. Most of them correspond to the arguments passed to the `show` method's callback.

```

Comment:
@jeffreylees Quick question before I merge: do you really want this as 'authenticated' or did you meant `authenticated`?